flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceTest.java

```diff
@@ -135,7 +135,7 @@ public void testGetAddress() {
 	 * @throws ExecutionException
 	 * @throws InterruptedException
 	 */
-	@Test(timeout = 1000)
+	@Test
```

Comment:
OK, never noticed this problem, but will check it the next time. Maybe we can improve something. ;-) I'll leave the JUnit timeout and increase it to a higher value like 60000. I think that's a good compromise?